app/js/actions/QuestionActions.js

```diff
@@ -22,6 +22,13 @@ QuestionActions = {
       data: data
     });
   },
+  recieveUserQuestions: data => {
+    waitForQuestionStore();
+    AppDispatcher.dispatch({
+      actionType: ZhishiConstants.RECEIVE_USER_QUESTIONS,
+      data: data
+    });
+  },
```

Comment:
Nice! Very minor, could you add the brackets to ` recieveUserQuestions: data => { ` so that the entire codebase looks uniform? i.e.,
` recieveUserQuestions: (data) => {`